assets/scss/_component-examples.scss

```diff
@@ -285,6 +285,10 @@
   }
 }
 
+.modal.show {
+  z-index: 1072;
```

Comment:
We'll also need to move the backdrop, right? The navbar doesn't need to be above the modal and it's backdrop here.